SplashScreen/app/src/main/res/values/themes.xml

```diff
@@ -36,17 +36,31 @@
     </style>
 
     <!--
-        The activity theme with animated icon in the splash screen.
+        The splash screen theme without any customization.
+        On API level 31 and above, this shows the app icon on the splash screen.
+        On API level 30 and below, this shows a default icon from the library.
+        See "Theme.App.SplashScreen.Animated" below for the backward-compatible splash screen.
     -->
-    <style name="Theme.SplashScreen.Animated">
-        <!-- The animated icon to be displayed at the center of the splash screen -->
-        <item name="android:windowSplashScreenAnimatedIcon">@drawable/ic_hourglass_animated</item>
+    <style name="Theme.App.SplashScreen" parent="Theme.SplashScreen">
+        <!-- The 'core-splashscreen' switches the theme back to this after splash screen. -->
+        <item name="postSplashScreenTheme">@style/Theme.App</item>
+    </style>
+
+    <!--
+        The splash screen theme with animated icon in the splash screen.
+    -->
+    <style name="Theme.App.SplashScreen.Animated" parent="Theme.SplashScreen">
```

Comment:
Just a suggestion, but to showcase that we can also set a custom background under the icon, you can use the Theme.SplashScreen.IconBackground theme, and set the windowSplashScreenIconAttribute

Reply:
Done.